docs/_autosummary/toqito.channel_metrics.completely_bounded_spectral_norm.rst

```diff
@@ -0,0 +1,6 @@
+﻿toqito.channel\_metrics.completely\_bounded\_spectral\_norm
+===========================================================
+
+.. currentmodule:: toqito.channel_metrics
+
+.. autofunction:: completely_bounded_spectral_norm
```

Comment:
@vprusso `autodocsummary` is adding these extra files because they don't exist. For this file, there should have been something at `docs/_autosummary/toqito.channel_metrics.completely_bounded_spectral_norm.rst`. 

Should I keep these or undo this commit?

https://github.com/vprusso/toqito/blob/master/docs/_autosummary/toqito.channel_metrics.completely_bounded_spectral_norm.rst